install/image-references

```diff
@@ -0,0 +1,8 @@
+kind: ImageStream
```

Comment:
we need to make sure install/ content is put on manifests/ within the docker image https://docs.google.com/document/d/1CGZVEyuloZ9oD4NUArW6dnEpi0WFc6BP2tqPSwFZTCY/edit?ts=5b916ae2#heading=h.1zgrwxmpgxbr

Reply:
sure, will update